actions/treasures.go

```diff
@@ -28,21 +31,29 @@ type treasureRes struct {
 	Success string `json:"success"`
 }
 
-func unixMilli(t time.Time) int64 {
-	return t.Round(time.Millisecond).UnixNano() / (int64(time.Millisecond) / int64(time.Nanosecond))
+// Constant API connection to IOTA
+var IOTA_API *giota.API
+
+func init() {
+	host_ip := os.Getenv("HOST_IP")
+	if len(host_ip) >= 0 {
+		host_ip = strings.Trim(host_ip, "http://")
+		host_ip = strings.Trim(host_ip, "https://")
+	} else {
+		host_ip = "localhost"
+	}
+	provider := fmt.Sprintf("http://%s:14265/", host_ip)
+	IOTA_API = giota.NewAPI(provider, nil)
```

Comment:
In a future PR, we should pull everything that touches iota into the iota_wrappers file, so we're not spreading iota stuff through the code base.